app/controllers/rooms_controller.rb

```diff
@@ -0,0 +1,38 @@
+class  RoomsController < ApplicationController
+
+  def index
+
```

Comment:
For the room index, what you'd need in order to get appropriate data would be: `@rooms = Room.all`

From there, you could iterate through these rooms in the view and for each one, call a Room model instance method to be able to get the count of guests that have stayed in the room (i.e. something like `room.get_guest_count` and the query in the Room model method would be `self.guests.distinct.count`)